### PR TITLE
feat(setup docker): Setup docker for backend

### PR DESCRIPTION
**Pull request by TuyishimireEric**
#### What does this PR do?

Docker setup

#### Description of Task to be completed?

Using docker compose, set up a ymal file that will run two services

express server
postgress db.
Note: there is nothing to worry about how render will run the app. this is only valid when we run locally, render will run the application in normal way

#### How should this be manually tested?

- download and install Docker desktop
- run `docker-compose up --build` 
- check the [swagger documentation ](http://localhost:3000/swagger/) and test the endpoints

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

- id of the project #187300140

### CODE REVIEW

File: Docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - "3000:3000"
+    env_file:
+      - .env
+    depends_on:
+      - db
+    environment:
+      NODE_ENV: development
+      DATABASE_PORT: 5432
+      DIALECT: postgres
+      JWT_SECRET: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6Im5hZGluZSIsImlhdCI6MTUxNjIzOTAyMn0.DBRiSfpqdyELAf8IjmFXHUH9AAUWX04hB2Xp_FRHa5E
```
Comment (fab-ryan) [inline Docker-compose.yml]: why this token ?

**Review by fab-ryan — CHANGES_REQUESTED** [review]
change that or remove them because we already have them

File: Docker-compose.yml (hunk 1 above)
Comment (TuyishimireEric) [reply Docker-compose.yml]: Let me remove it, it is not necessarily

**Review by fab-ryan — APPROVED** [review]
Good work

**Review by RWEMAREMY — APPROVED** [review]
good job

**Review by Sophie20202 — APPROVED** [review]

**Review by Mukunzijames — APPROVED** [review]

**Review by manzidavidrw — APPROVED** [review]

**Pull request merged**